src/cljs/ataru/virkailija/editor/components/text_component.cljs

```diff
@@ -22,68 +22,114 @@
 (defn- get-val [event]
   (-> event .-target .-value))
 
-(defn- decimal-places-selector [component-id path]
-  (let [decimal-places (subscribe [:editor/get-component-value path :params :decimals])
-        component-locked?   (subscribe [:editor/component-locked? path])
-        min-value      (subscribe [:editor/get-range-value component-id :min-value path])
-        max-value      (subscribe [:editor/get-range-value component-id :max-value path])
-        min-invalid?   (subscribe [:state-query [:editor :ui component-id :min-value :invalid?]])
-        max-invalid?   (subscribe [:state-query [:editor :ui component-id :max-value :invalid?]])
-        min-id         (util/new-uuid)
-        max-id         (util/new-uuid)
-        format-range   (fn [value]
-                         (string/replace (string/trim (or value "")) "." ","))]
+(defn- numeerisen-kentän-muoto [{:keys [path]}]
+  (let [decimal-places    (subscribe [:editor/get-component-value path :params :decimals])
+        component-locked? (subscribe [:editor/component-locked? path])]
```

Comment:
Onko tosiaan niin että `path` -property yllä ei muutu, eli onko tarkoitus että on render-funktion ulkopuolella?

Reply:
Näin rakenne oli alunperin `decimal-places-selector` funktiossa, joten se toiminee.